website/docs/docs/quickstarts/dbt-cloud/databricks-qs.md

```diff
@@ -0,0 +1,286 @@
+---
+title: "Quickstart for dbt Cloud and Databricks"
+description: "Quickstart for dbt Cloud and Databricks."
+id: "databricks"
+sidebar_label: "Databricks quickstart"
+---
+For the Databricks project in the quickstart guide, you'll learn how to use dbt Cloud with Databricks.
```

Comment:
Think i commented on this in at least one other guide, but this sentence feels confusing (and maybe even redundant given the content of the next paragraph)